search/search.py

```diff
@@ -46,6 +46,20 @@ def trigram_search(query: str, queryset, fields: List[str]):
     return queryset.annotate(similarity=trigram_query).order_by('-similarity')
 
 
+def field_type(model, field):
```

Comment:
up to now we had convenient to use verbs as the first word of a function name.
So, let's call it `get_field_type` or change this convenient.
The both is ok for me, but convenient preserving should just cheaper.

Decide it plz